snapshot/cli/cli.go

```diff
@@ -184,31 +185,44 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 //	--basis="<rev>" --ref="master" --ttl="336h" --bundlestore_url="http://localhost:9094/bundle"
 // Stdout: "http://localhost:9094/bundle/bs-<rev>-master.bundle"
 type createGitBundleCommand struct {
-	file  string
-	basis string
-	ref   string
-	ttld  time.Duration
-	keep  bool
+	basis  string
+	ref    string
+	ttld   time.Duration
+	output string
```

Comment:
is this output type?  perhaps some comments on the fields:
basis is git commit sha (as returned in 'git log' commands),
rev is ?
ttld is how long (in hours) the snapshot should be managed by bundlestore
output is ?